packages/clerk-js/src/ui/components/OrganizationList/OrganizationListPage.tsx

```diff
@@ -121,6 +121,11 @@ const OrganizationListPageList = (props: { onCreateOrganizationClick: () => void
   const handleCreateOrganizationClicked = () => {
     props.onCreateOrganizationClick();
   };
+
+  // Solve weird bug with swr while running unit tests
+  const userInvitationsData = userInvitations.data?.filter(a => !!a);
+  const userSuggestionsData = userSuggestions.data?.filter(a => !!a);
```

Comment:
Generally, it's a bad practice to alter the way your code works in favour of the tests. 😿

Reply:
Totally agree, for some reason only in test and not runtime in a browser `swr` will populate data with `[undefined]` and `total_count` would still be `0`. I have not figured out why this is happening, I have added a note in order to come back later and investigate deeper.